build.linux32x86/editpharoinstall.sh

```diff
@@ -11,8 +11,11 @@ if [ "$1" = -source ]; then
 else
 	SourceFile="PharoV50"
 fi
-SOURCE=../../sources/$SourceFile.sources
-test -f $SOURCE || SOURCE=../../../sources/$SourceFile.sources
```

Comment:
Maybe a copy and paste error? There used to be a [`sources` directory](http://www.squeakvm.org/svn/squeak/branches/Cog/sources/) in the root of the SVN repository, but we removed it when we moved to Git.